lib/analysis/index.js

```diff
@@ -0,0 +1,17 @@
+const utils = require("../utils");
+const {call_analysis_api} = require('../api_client/call_analysis_api');
+
+function analyze(uri, analysis_type, options = {}, callback) {
```

Comment:
Don't we want it to be called `analyze_uri`?
Soon we might have analyze by asset id or others.

Reply:
Good idea, changed the name.